DOCS-120, DOCS-121, DOCS-123 Fixes (#1580)
100mslive/100ms-docs: docs/server-side/v2/api-reference/legacy-api-v1/room.mdx

| template <br/>       | `string` | [Template](/concepts/v2/concepts/templates-and-roles/overview) of the room. You can get template name from room details section on dashboard                                                                                                                                                                                                                                                                                                                                 | No       |
| recording_info <br/> | `object` | Object of type `recording_info`. This object contains information for enabling recording/setting storage location for recordings.                                                                                                                                                                                                                                                                                                                           | No       |

> | region <br/>         | `string` | Region in which you want to create room. <br/><br/> `eu` - European Union <br/><br/> `in` - India <br/><br/> `us` - United States <br/><br/><br/> `auto` Automatic region selection (default, and in case this parameter is not provided)                                                                                                                                                                                                                        | No       |

Reviewer: ```suggestion
| region <br/>         | `string` | Region in which you want to create a room. <br/><br/> `eu` - European Union <br/><br/> `in` - India <br/><br/> `us` - United States <br/><br/><br/> `auto` Fallback to region of the template (default value)                                                                                                                                        | No       |
```
